snippets/header-dropdown-menu.liquid

```diff
@@ -12,7 +12,10 @@
         {%- if link.links != blank -%}
           <header-menu>
             <details id="Details-HeaderMenu-{{ forloop.index }}">
-              <summary class="header__menu-item list-menu__item link focus-inset">
+              <summary
+                id="HeaderMenu-{{ link.handle }}"
```

Comment:
Is there potential/likelihood to have multiple elements with the same id, since we render both the mobile drawer and dropdown menu? I believe that's invalid HTML and will show a console error.

If so you could add it as a class or data attribute instead - I think it achieves the same goal of being able to target the elements with CSS. Alternatively you could have different prefix for each menu (e.g. `HeaderDrawer`) which would make it even more targetable but a little more CSS.

The naming convention looks good to me 👍